atime_wasm.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// http://golang.org/src/os/stat_wasm.go
```

Comment:
URL doesn't exist, I have the correct one in my times library:
https://github.com/djherbis/times/blob/master/times_js.go

You should copy some of the impl from there too, since it looks like yours doesn't handle Nsec.

Reply:
done!

- fix url
- handle nsec